gptcli/providers/azure_openai.py

```diff
@@ -0,0 +1,12 @@
+import openai
+from openai import AzureOpenAI
+from gptcli.providers.openai import OpenAICompletionProvider
+
+
+class AzureOpenAICompletionProvider(OpenAICompletionProvider):
+    def __init__(self):
+        self.client = AzureOpenAI(
```

Comment:
let's call `super().__init__(self)` here - in case we ever decide to do some initialization in the parent class' `__init__` other than setting the client